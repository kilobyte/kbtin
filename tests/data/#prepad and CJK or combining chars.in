#showme NORMAL
#prepad tmp 3 blah;#showme A($tmp)
#prepad tmp 5 blah;#showme B($tmp)
#showme NORMAL + COMBINING
#prepad tmp 3 b́ĺáh́;#showme A($tmp)
#prepad tmp 5 b́ĺáh́;#showme B($tmp)
#showme CJK
#prepad tmp 3 ァアィイ;#showme A($tmp)
#prepad tmp 9 ァアィイ;#showme B($tmp)
#showme CJK + COMBINING
#prepad tmp 3 ァ́ア́ィ́イ́;#showme A($tmp)
#prepad tmp 9 ァ́ア́ィ́イ́;#showme B($tmp)
