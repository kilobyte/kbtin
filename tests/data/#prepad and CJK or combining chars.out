NORMAL
A(bla)
B( blah)
NORMAL + COMBINING
A(b́ĺá)
B( b́ĺáh́)
CJK
A( ァ)
B( ァアィイ)
CJK + COMBINING
A( ァ́)
B( ァ́ア́ィ́イ́)
